.github/workflows/pr-title-semantic-lint.yml

```diff
@@ -0,0 +1,16 @@
+name: PrTitleSemanticLint
+on:
+  pull_request:
+    branches: [master]
+    types: [opened, edited, synchronize, reopened]
+jobs:
+  Lint:
+    runs-on: ubuntu-20.04
+    env:
+      GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3
+      - name: Ensure PR Title is Semantic
+        run: |
+          npm ci
+          npx @coveo/is-pr-title-semantic
```

Comment:
I wonder if we could just add an `action.yml` file to this package in the coveo/github-actions repo so that we don't need a package.json

Reply:
tbh, quite frequent nowadays to have package json for scripts and shenanigans, because a good ecosystem has grown there.
I think we would end up with that eventually regardless

Reply:
Yeah, I agree with @louis-bompart. As long as the `package.json` does not affect developers, I don't really mind.